roles/iptables/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Install iptables-persistent package
+  become: yes
+  apt: 
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - iptables-persistent
+  register: install_iptables_persistent
+- name: Copy iptables templates
+  become: yes
+  template: 
+    src: "iptables_v4.j2"
+    dest: "/etc/iptables/rules.v4"
+    backup: no
+    owner: root
+    group: root
+    mode: '0644'
+  notify: restart iptables-persistent
+  when: install_iptables_persistent.changed == True
```

Comment:
better to check dest file instead of installation, since package could be updated

Reply:
as comment in INSTRM-174, seems no way to detect updated but not installed. go to a way of checking dest file.